packages/ui-utils/src/types.ts

```diff
@@ -266,6 +284,14 @@ An optional object to be passed to the API endpoint.
   body?: object;
 };
 
+export interface URLFile {
+  name: string;
+  type: string | undefined;
+  url: string;
+}
+
+export type URLFileList = Array<URLFile>;
```

Comment:
No need for an explicit type imo. I'd prefer to avoid exposing it (users will use it and then we have to manage it for backwards compat). We could just inline into files below:

`files?: FileList | Array<MessageFile>;

Reply:
done!